presto-main/src/main/java/io/prestosql/security/AccessControlManager.java

```diff
@@ -118,11 +120,14 @@ public void loadSystemAccessControl()
     {
         log.info("-- Loading system access control --");
 
-        File configFile = CONFIG_FILE.getAbsoluteFile();
-        if (!configFile.exists()) {
-            setSystemAccessControl(AllowAllSystemAccessControl.NAME, ImmutableMap.of());
-            log.info("-- Loaded system access control %s --", AllowAllSystemAccessControl.NAME);
-            return;
+        File configFile = this.configFile;
+        if (configFile == null) {
+            if (!CONFIG_FILE.exists()) {
+                setSystemAccessControl(AllowAllSystemAccessControl.NAME, ImmutableMap.of());
+                log.info("-- Loaded system access control %s --", AllowAllSystemAccessControl.NAME);
+                return;
+            }
+            configFile = CONFIG_FILE;
```

Comment:
Retain `CONFIG_FILE.getAbsoluteFile()`, because this path gets printed later on and it should be clear where is the file being mentioned in the message. Relative path `etc/xxx` is confusing.

(In fact, this `getAbsoluteFile()` should be applied to explicitly configured file as well, in case a relative path was used and eg we later fail with file not found)